src/config.js

```diff
@@ -4,6 +4,8 @@ import winston from 'winston';
  * BOT CONFIGURATION:
  */
 
+export const HIPCHAT_CALLBACK_URL = process.env.HIPCHAT_CALLBACK_URL || '';
```

Comment:
Yeah I prefer to keep env variables out when possible. When deploying to Heroku, for example, you only need to optionally set a username and password for the admin panel.

The benefit of utilizing our own setup flow is that we can point users in the correct direction better and reduce the risk of them bailing.